models/contacts.js

```diff
@@ -1,14 +1,131 @@
-// const fs = require('fs/promises')
+const { readFile, writeFile } = require("fs/promises");
+const Joi = require("joi");
+const path = require("path");
+const { v4: uuidv4 } = require("uuid");
 
-const listContacts = async () => {}
 
-const getContactById = async (contactId) => {}
+const fileName = "contacts.json";
+const contactsPath = path.join(__dirname, fileName);
 
-const removeContact = async (contactId) => {}
+const getContactsArray = async () => {
+  return await readFile(contactsPath, "utf-8")
+    .then((data) => JSON.parse(data))
+    .catch((error) => console.log(error.message));
```

Comment:
Może lepiej użyć tutaj bloku try catch? 
Ta linijka kodu w którym robisz catch ona pisze do terminala że jest jakiś błąd natomiast prawdopodobnie użytkownik frontendu otrzyma że wszystko jest w porządku. Ten błąd nie zostanie przesłany dalej a żeby został trzeba użyć Throw. 

Możesz to łatwo sprawdzić jak się zachowa w przypadku błędu Wystarczy że `JSON.parse('dsdss')`